docs/framework/wcf/feature-details/messaging-activities.md

```diff
@@ -43,20 +43,20 @@ Messaging activities allow workflows to send and receive WCF messages. By adding
   
- Because of the long-running nature of workflows, it is important for the duplex pattern of communication to also support long-running conversations. To support long-running conversations, clients who initiate the conversation must provide the service with an opportunity to call it back at a later time when the data becomes available. For example, a purchase order request is submitted for manager approval, but it might not be processed for a day, a week, or even a year; the workflow that manages the purchase order approval must know to resume after the approval is given. This pattern of duplex communication is supported in workflows using correlation. To implement a duplex pattern, use <xref:System.ServiceModel.Activities.Send> and <xref:System.ServiceModel.Activities.Receive> activities. On the <xref:System.ServiceModel.Activities.Receive> activity, initialize a correlation using the special key value of <xref:System.ServiceModel.Activities.CorrelationHandle.CallbackHandleName%2A>. On the <xref:System.ServiceModel.Activities.Send> activity set that correlation handle as the <xref:System.ServiceModel.Activities.Send.CorrelatesWith%2A> property value. [!INCLUDE[crdefault](../../../../includes/crdefault-md.md)][Durable Duplex](../../../../docs/framework/wcf/feature-details/durable-duplex-correlation.md).  
+ Because of the long-running nature of workflows, it is important for the duplex pattern of communication to also support long-running conversations. To support long-running conversations, clients who initiate the conversation must provide the service with an opportunity to call it back at a later time when the data becomes available. For example, a purchase order request is submitted for manager approval, but it might not be processed for a day, a week, or even a year; the workflow that manages the purchase order approval must know to resume after the approval is given. This pattern of duplex communication is supported in workflows using correlation. To implement a duplex pattern, use <xref:System.ServiceModel.Activities.Send> and <xref:System.ServiceModel.Activities.Receive> activities. On the <xref:System.ServiceModel.Activities.Receive> activity, initialize a correlation using the special key value of <!--<xref:System.ServiceModel.Activities.CorrelationHandle.CallbackHandleName%2A>-->`System.ServiceModel.Activities.CorrelationHandle.CallbackHandleName`. On the <xref:System.ServiceModel.Activities.Send> activity set that correlation handle as the <xref:System.ServiceModel.Activities.Send.CorrelatesWith%2A> property value. [!INCLUDE[crdefault](../../../../includes/crdefault-md.md)][Durable Duplex](../../../../docs/framework/wcf/feature-details/durable-duplex-correlation.md).  
```

Comment:
This was entered into the Migration Project for investigation, but I'll go ahead and add the "zz" just in case the note there is accidentally deleted.